src/tiledbsoma_ml/_distributed.py

```diff
@@ -14,38 +14,38 @@
 
 
 def get_distributed_world_rank() -> Tuple[int, int]:
-    """Return tuple containing equivalent of |torch.distributed| world size and rank."""
-    world_size, rank = 1, 0
+    """Return tuple containing equivalent of |torch.distributed| rank and world size."""
+    rank, world_size = 0, 1
     if "RANK" in os.environ and "WORLD_SIZE" in os.environ:
-        world_size = int(os.environ["WORLD_SIZE"])
         rank = int(os.environ["RANK"])
+        world_size = int(os.environ["WORLD_SIZE"])
     elif "LOCAL_RANK" in os.environ and "WORLD_SIZE" in os.environ:
         # Lightning doesn't use RANK! LOCAL_RANK is only for the local node. There
         # is a NODE_RANK for the node's rank, but no way to tell the local node's
         # world. So computing a global rank is impossible(?).  Using LOCAL_RANK as a
         # proxy, which works fine on a single-CPU box. TODO: could throw/error
         # if NODE_RANK != 0.
-        world_size = int(os.environ["WORLD_SIZE"])
         rank = int(os.environ["LOCAL_RANK"])
+        world_size = int(os.environ["WORLD_SIZE"])
     elif torch.distributed.is_initialized():
         world_size = torch.distributed.get_world_size()
         rank = torch.distributed.get_rank()
 
-    return world_size, rank
+    return rank, world_size
```

Comment:
the re-ordering of the return values should require a change in method name.  It is "get world rank".

If you reorder the return value, suggest also renaming to `get_distributed_rank_world`.  Ditto for `get_worker_...`